refinery/core/search_indexes.py

```diff
@@ -66,14 +74,20 @@ def prepare_submitter(self, object):
 
         for contact in investigation.contact_set.all():
             submitters.append(
-                "{}, {}".format(contact.last_name, contact.first_name)
+                unicode("{}, {}").format(
```

Comment:
We should avoid `unicode()` because it does not exist in Python 3. Either add `u''` prefix or import `unicode_literals`: http://python-future.org/compatible_idioms.html#unicode
Thoughts?